### ISSUE TITLE
Provide a browser build of lodash-migrate

### ISSUE THREAD

**Issue by Delapouite** [lodash/lodash-migrate#6]
Hi

I'm currently using lodash on a client side app, by simple adding it to the page with a script tag.
Do you plan to build a lodash-migrate.js that could be used like this : 

```html
<script src="lodash.js"></script>
<script src="lodash-migrate.js"></script>
```

**Comment by d10** [comment]
Not too sure if we want to release this for browsers as well, but for now you can Browserify the module.

**Comment by jdalton** [comment]
Yap for now try Browserify. We've had a couple report success using it with lodash-migrate.

**Comment by bbottema** [comment]
@jdalton 

This is a real issue for us.

We're using requirejs already instead of browserify and have a huge codebase using lodash 3.8. This lodash 4 update without proper migration path proves unfeasible. Lodash-migrate is not working for us in the browser.

I already tried creating a browserified 'compiled' version of lodash-migrate and adding a requirejs _define_ but it doesn't seem to work right now.

If we had some version in between with deprecation markings, at least we could have migrated gracefully.

**Comment by jdalton** [comment]
> We're using requirejs already instead of browserify and have a huge codebase using lodash 3.8 with several teams working on it. 

It would be browserifying as in bundled with everything else so not really for AMD.
I'm open to pull requests to enable AMD support.

**Comment by JasonMore** [comment]
@bbottema @jdalton I got it working in my project like this:

1) install browserify ```npm install -g browserify```
2) made a basic node app called lodash-migrate.js
3) ran browserify command line
4) added bundled script to my index.html file

lodash-migrate.js
```
var lodash = require('lodash');
require('lodash-migrate');
window._ = lodash;
```

run command line browserify
```browserify lodash-migrate.js -o bundle.js```

index.html
```
<script src="/bundle.js"></script>
```

**Comment by just-boris** [comment]
Also, there is an [online service](https://wzrd.in/) to build browserify bundles. I tried to build lodash-mirgate, but without luck:
https://wzrd.in/standalone/lodash-migrate

I happened because `lodash@3.10` in the devDependencies. Service installs only production dependencies.

@jdalton why don't you move it to `dependencies` section and make possible to grab `lodash-migrate` from there?

**Comment by jdalton** [comment]
> why don't you move it to dependencies section and make possible to grab lodash-migrate from there?

Because `lodash-migrate` works with whatever version of `lodash` you're using. Our unit tests use `lodash@3.10.1`.

**Comment by just-boris** [comment]
Well, then I published my build of `lodash-migrate+lodash@3.10.1`

https://github.com/just-boris/lodash3to4

You can install it via Bower and use in the browser without extra building.

**Comment by jdalton** [comment]
> Well, then I published my build of `lodash-migrate+lodash@3.10.1`

I'd rather you contribute it back to `lodash-migrate` instead of publishing another package.

**Comment by jdalton** [comment]
Send us a PR.
